pkg/motor/create_account.go

```diff
@@ -11,20 +11,42 @@ import (
 	"github.com/sonr-io/sonr/pkg/did/ssi"
 	"github.com/sonr-io/sonr/pkg/tx"
 	"github.com/sonr-io/sonr/pkg/vault"
+	"github.com/sonr-io/sonr/third_party/types/common"
 	mt "github.com/sonr-io/sonr/third_party/types/motor/api/v1"
 	rt "github.com/sonr-io/sonr/x/registry/types"
 )
 
 func (mtr *motorNodeImpl) CreateAccount(request mt.CreateAccountRequest) (mt.CreateAccountResponse, error) {
 	// create DSC and store it in keychain
+	mtr.triggerWalletCreateEvent(common.WalletEvent{
```

Comment:
We should call this method something other than `triggerWalletCreateEvent` if it's going to be used for any type of event. Like `triggerWalletEvent`.

Reply:
I can rename to this. its only called in `create_account` methods so i named it like that.